demos/index.html

```diff
@@ -14,19 +13,39 @@
         <script type="text/javascript" src="../src/js/dataEntry.js"></script>
         <script type="text/javascript" src="../src/js/dataEntryPanel.js"></script>
         <script type="text/javascript" src="../src/js/chartAuthoring.js"></script>
+        <script type="text/javascript" src="../src/js/d3ViewComponent.js"></script>
+        <script type="text/javascript" src="../src/js/legend.js"></script>
+        <script type="text/javascript" src="../src/js/pie.js"></script>
+        <script type="text/javascript" src="../src/js/pieChart.js"></script>
+        <script type="text/javascript" src="../src/js/d3Utils.js"></script>
+
+        <link rel="stylesheet" type="text/css" href="src/css/chartAuthoring-demo.css" />
 
         <title>Chart Authoring Tool</title>
     </head>
 
     <body>
-        <div id="floec-chartAuthoring"></div>
+        <div id="floec-chartAuthoring">
+            <div id="floec-chartAuthoring-pieChart"></div>
+            <div id="floec-chartAuthoring-dataEntryPanel"></div>
+        </div>
 
         <script type="text/javascript">
             floe.chartAuthoring("#floec-chartAuthoring", {
+                selectors: {
```

Comment:
I see, you do have a "selectors" option defined in the defaults block of ```floe.chartAuthoring```. We probably should match the html template selectors with selector names defined in there, to simplify our demo code by getting rid of this option.